subsys/bluetooth/audio/ascs.c

```diff
@@ -1640,14 +1640,22 @@ static ssize_t ascs_qos(struct bt_ascs *ascs, struct net_buf_simple *buf)
 
 		LOG_DBG("ase 0x%02x", qos->ase);
 
-		ase = ase_find(ascs, qos->ase);
-		if (!ase) {
+		if (!qos->ase || qos->ase > ASE_COUNT) {
 			ascs_cp_rsp_add(qos->ase, BT_ASCS_QOS_OP,
 					BT_ASCS_RSP_INVALID_ASE, 0x00);
 			LOG_WRN("Unknown ase 0x%02x", qos->ase);
 			continue;
 		}
 
+		ase = ase_find(ascs, qos->ase);
+		if (!ase) {
+			LOG_WRN("Invalid operation in state: %s",
+				bt_audio_ep_state_str(BT_AUDIO_EP_STATE_IDLE));
```

Comment:
Suggest to do
```suggestion
			LOG_DBG("Invalid operation for idle ASE");
```

To make it a bit more concise

Reply:
This of course applies too all these occurrences